docs/user-event/options.mdx

```diff
@@ -6,6 +6,13 @@ title: Options
 The following options allow to adjust the behavior of `user-event` APIs. They
 can be applied per [`setup()`](setup.mdx).
 
+### advanceTimers
+
+A function to advance your fake timers (if applicable), when we
```

Comment:
This could be me, but I had to read the sentence multiple times to understand it.
What do you think about the following?

```suggestion
A function to advance your fake timers (if applicable). Used when we
```

```suggestion
A function to advance your fake timers (if applicable). Useful when we
```

Reply:
"Useful" doesn't nail it. "Used" is okay I think. But maybe we should rephrase the whole sentence if it is hard to comprehend.

How about?
> If you are using fake timers, you need to advance your timers when we internally [delay](#delay) subsequent code.
> ```ts
> (delay: number) => Promise<void> | void
> ```

Reply:
I like that! Let's use that one.